src/features/jserrors/shared/cast-error.js

```diff
@@ -46,15 +46,15 @@ export function castPromiseRejectionEvent (promiseRejectionEvent) {
 
   if (canTrustError(promiseRejectionEvent.reason)) {
     try {
-      promiseRejectionEvent.reason.message = prefix + ': ' + promiseRejectionEvent.reason.message
-      return castError(promiseRejectionEvent.reason)
+      if (!promiseRejectionEvent.reason.message.startsWith(prefix)) promiseRejectionEvent.reason.message = prefix + promiseRejectionEvent.reason.message
     } catch (e) {
-      return castError(promiseRejectionEvent.reason)
+      // failed to modify the message, do nothing else
     }
+    return castError(promiseRejectionEvent.reason)
   }
 
   const error = castError(promiseRejectionEvent.reason)
-  error.message = prefix + ': ' + error?.message
+  if (!((error?.message || '').startsWith(prefix))) error.message = prefix + error?.message
```

Comment:
I think `error` is defined at this point, so we can drop the `?`s

```suggestion
  if (!((error.message || '').startsWith(prefix))) error.message = prefix + error.message
```